Doc/library/zipfile.rst

```diff
@@ -183,6 +184,9 @@ ZipFile Objects
       Previously, a plain :exc:`RuntimeError` was raised for unrecognized
       compression values.
 
+   .. versionchanged:: 3.6.2
```

Comment:
Given that this landed in the 3.6 branch, doesn't that mean that the upcoming 3.6.1 will contain this?